addons/forum-quote-code-beautifier/addon.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Forums Quote & Code Beautifier",
```

Comment:
Name capitalization?

Reply:
Also, a name like "Customized quotes on forums" would better explain this addon's purpose.

Reply:
Well, I think the name should be ok the short side than the descriptive side. It's just the name, not the description.

@GrahamSH-LLK sorry for late response on this one